content/sidebar/recommended_docs_forms.md

```diff
@@ -2,7 +2,7 @@ title: Рекомендуемые формы документов
 slug: recommended_docs_forms
 category: other
 tags: [graduate]
-sidebar_index: 2
+sidebar_index: 4
```

Comment:
You should not have changed the file. Change `content/sidebar/documents.md` instead.